styles.css

```diff
@@ -0,0 +1,72 @@
+html {
+   box-sizing: border-box;
+   padding: 0;
+   margin: 0;
+}
+body{
+   background-color:aqua;
+   text-align: center;
+   width: 100%;
+}
+
+#video {
+  width: 350px;
+  height: 40%;
+  margin: 0 auto;
+}
+
+#img-header {
+   width: 150px;
+
+}
+
+#header {
+   position: fixed;
+   top: 0;
+   min-height: 75px;
+   padding: 0px 20px;
+   display: flex;
+   justify-content: space-around;
+   align-items: center;
+   background-color: beige;
+   width: 100%;
+   margin: 0;
+   padding-left: 0;
+
+}
+
+ul {
+   display: flex;
+   justify-content: space-around;
+   width: 35vw;
+   align-content: flex-end;
+   flex-direction: row;
+
+}
+
+li{
+  text-decoration: none; 
+  list-style: none;
+}
+
+a{
+   text-decoration: none; 
+   list-style: none;
+   color:black;
+ }
+
+#nav-link {
+   justify-content: left;
+}
+
+
+h1 {
+   margin-top: 150px;
+   
+}
+
+footer {
+   display: flex;
+   justify-content: space-around;
+   
+}
```

Comment:
- [ ] Good job praising the student's hard work. but I think our suggestion for change should be more precise and clear to the student.
- [ ] And as I mentioned earlier that instead of adding all the lines of the file in inline comment, a code reviewer should only add the lines of code in an inline comment where a problem occurs. so that it would be easier for the student to understand the specific point in the code and to fix that.
- [ ] I would also suggest here to let the student know the reason for required changes.